### ISSUE TITLE
Startup usage issue

### ISSUE THREAD

**Issue by glensc** [tanimislam/howdy#9]
How do I start using this. thing?

From https://tanimislam.github.io/howdy/howdy-config/howdy_settings_config.html#howdy-settings-configuration

I read out that I should first launch `howdy_config_gui`, but it gives me error and aborts:

```
$ howdy_config_gui
ERROR, NO TMDB API CREDENTIALS FOUND
```

**Comment by tanimislam** [comment]
@glensc it is worth a deeper-dive discussion on how to get started there.

Setting up the proper credentials among 12 or so services is a mess. I have a worked-for-me-alone in 2019 section of the howdy documentation on getting all the credentials working. It was a mess then that I somehow got working and then forgot.

**Comment by glensc** [comment]
can you give me introduction what should I do? I'll  give you feedback of the progress :)

I haven't yet figured is this program any use for me, since can't get it up

**Comment by tanimislam** [comment]
At this point, I can DM you (directly) my set of credentials. You have to install the `app.db` into `~/.config/howdy`, as `~/.config/howdy/app.db`

**Comment by glensc** [comment]
no. don't send me your credentials. what is `app.db`, how it's created?
